templates/internal/appName/rpc/rpc.go.tpl

```diff
@@ -181,8 +181,8 @@ func (gs *GRPCService) StartServers(ctx context.Context, servers *Servers, opts.
 	// end gRPC RPCs injected by modules
 	{{- end }}
 
-	// Register default server
-	api.Register{{ title $pkgName }}Server(s, rpcserver{servers.DefaultServer})
+	// Register default server, title function won't work well when use underscore, so we make it dash first
+	api.Register{{ $pkgName | replace "_" "-" | title | replace "-" "" }}Server(s, rpcserver{servers.DefaultServer})
```

Comment:
@ygcao, I believe a few more place also need to have similar changes. 
One example:
https://github.com/getoutreach/stencil-golang/blob/3f5fd3cef6ff980546f2a6b859ed4588d88ec812/templates/api/rpc/client.go.tpl#L75
And also L83, L105, L114.

Reply:
you could be right. I didn't try the Go client yet thus didn't bother by it yet. will add same logic there too. maybe it's time for making it another function to use in needed places then. something like `goTitleCaseName`

Reply:
I finally found I indeed has the go client code generated, and there were fine somehow. Looks doesn't need a fix, do you have example broken generated codes?
<img width="900" alt="image" src="https://user-images.githubusercontent.com/6810045/226438574-0bdcb000-9274-4000-a399-63f828e11a80.png">

Reply:
Never mind, found since the file was `Static`, it didn't do refresh for the new runs of `stencil`. They indeed need a fix too.

Reply:
Thanks for the confirmation. FYI, this is a test from my side: 
<img width="569" alt="Screenshot 2023-03-20 at 11 57 51 AM" src="https://user-images.githubusercontent.com/88365099/226439345-548bfe7f-5808-40d7-9593-92eee844a2a8.png">